commit 5535bdb75ed897160af0337961898c5e456b16c5
Author:     Yuan-Ting Hsieh (謝沅廷) <yuantingh@nvidia.com>
AuthorDate: Mon Mar 20 16:23:33 2023 -0700
Commit:     GitHub <noreply@github.com>
CommitDate: Mon Mar 20 16:23:33 2023 -0700

    Enhance job meta validator (#1555)
    
    * Enhance job meta validator
    
    * Fix typo
---
 nvflare/app_common/job_schedulers/job_scheduler.py | 7 ++-----
 1 file changed, 2 insertions(+), 5 deletions(-)

diff --git a/nvflare/app_common/job_schedulers/job_scheduler.py b/nvflare/app_common/job_schedulers/job_scheduler.py
index cfb89eac..02898776 100644
--- a/nvflare/app_common/job_schedulers/job_scheduler.py
+++ b/nvflare/app_common/job_schedulers/job_scheduler.py
@@ -21,14 +21,11 @@ from nvflare.apis.event_type import EventType
 from nvflare.apis.fl_component import FLComponent
 from nvflare.apis.fl_constant import FLContextKey
 from nvflare.apis.fl_context import FLContext
-from nvflare.apis.job_def import ALL_SITES, Job, JobMetaKey, RunStatus
+from nvflare.apis.job_def import ALL_SITES, SERVER_SITE_NAME, Job, JobMetaKey, RunStatus
 from nvflare.apis.job_def_manager_spec import JobDefManagerSpec
 from nvflare.apis.job_scheduler_spec import DispatchInfo, JobSchedulerSpec
 from nvflare.apis.server_engine_spec import ServerEngineSpec
 
-SERVER_SITE_NAME = "server"
-
-
 SCHEDULE_RESULT_OK = 0  # the job is scheduled
 SCHEDULE_RESULT_NO_RESOURCE = 1  # job is not scheduled due to lack of resources
 SCHEDULE_RESULT_BLOCK = 2  # job is to be blocked from scheduled again due to fatal error
@@ -152,7 +149,7 @@ class DefaultJobScheduler(JobSchedulerSpec, FLComponent):
             else:
                 resource_reqs[site_name] = {}
 
-        job_participants = [fl_ctx.get_identity_name(default="server")]
+        job_participants = [fl_ctx.get_identity_name(default=SERVER_SITE_NAME)]
         job_participants.extend(applicable_sites)
 
         fl_ctx.set_prop(FLContextKey.CURRENT_JOB_ID, job.job_id, private=True)
